<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tutordashboard</class>
 <widget class="QDialog" name="tutordashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
  </property>
  <widget class="QTextBrowser" name="top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>192</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; color:#354680;&quot;&gt;Hello TUTOR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>931</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="RequestTab">
    <property name="styleSheet">
     <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
    </property>
    <attribute name="title">
     <string>Request</string>
    </attribute>
    <widget class="QFrame" name="request">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="request_out"/>
      </item>
      <item>
       <widget class="QGroupBox" name="request_input">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="req_in">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="placeholderText">
            <string>Request ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="acceptreq_btn">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Accept Request</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="RatingsTab">
    <property name="styleSheet">
     <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
    </property>
    <attribute name="title">
     <string>Ratings</string>
    </attribute>
    <widget class="QGroupBox" name="Ratings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextBrowser" name="ratings_out"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="SessionsTab">
    <property name="styleSheet">
     <string notr="true">QPushButton { color: rgb(32, 72, 175); background-color: rgb(176, 230, 255)}
QDialog{background-color: rgb(218, 241, 255)}QTextBrowser{background-color: rgb(218, 241, 255)}QLineEdit{background-color: white}</string>
    </property>
    <attribute name="title">
     <string>Sessions</string>
    </attribute>
    <widget class="QFrame" name="sessions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>551</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextBrowser" name="sessions_out"/>
      </item>
      <item>
       <widget class="QGroupBox" name="session_in">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="session_id">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="placeholderText">
            <string>Session ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="endsession_btn">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>End Session</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="refresh_btn">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
